Applications/DataExplorer/DataView/CMakeLists.txt

```diff
@@ -134,7 +134,7 @@ add_library(QtDataView
 )
 
 target_link_libraries(QtDataView
-    PUBLIC GeoLib MeshGeoToolsLib MeshLib QtBase VtkVis
+    PUBLIC GeoLib MeshGeoToolsLib MeshLib QtBase # VtkVis
```

Comment:
Drop them, don't comment, but probably you would do this anyway, just reminding...